active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,565 @@
+
+### Add an offset 
+
+You can control how far a pointing teaching tip is set apart from its target and how far a non-pointing teaching tip is set apart from the the edges of the XamlRoot using the TargetOffset property. Like [Margin](https://docs.microsoft.com/en-us/dotnet/api/system.windows.frameworkelement.margin?view=netframework-4.7.2), TargetOffset has four values – left, right, top, and bottom – so only the relevant values are used. For example, if the target is left of the teaching tip, the TargetOffset.Left will be used. If the teaching tip has no target, the TargetOffset’s Right and Bottom will be used to space it from the bottom right corner of the XamlRoot.  
+
+The following example shows a non-pointing tip with the TargetOffset’s Left/Top/Right/Bottom all set to 80.
+
+XAML
+```XAML
+<Button x:Name="SaveButton" Content="Save" />
+
+<muxc:TeachingTip x:Name="AutoSaveTip"
+    Title="Saving automatically"
+    Subtitle="We save your changes as you go - so you never have to."
+    TargetOffset="80">
+</muxc:TeachingTip>
+```
+
+![A sample app with a teaching tip positioned toward, but not fully against, the bottom right corner. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipTargetOffsetSample.jpg)
+
+
+### Add content
+
+Content can be added to a teaching tip using the Content property. If there is more content to show than what the size of a teaching tip will allow, a scrollbar will be automatically enabled to allow a user to scroll the content area. 
+
+XAML
+```XAML
+<Button x:Name="SaveButton" Content="Save">
+    <Button.Resources>
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Target="{x:Bind SaveButton}"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to.">
+                <StackPanel>
+                    <CheckBox x:Name="HideTipsCheckBox" Content="Don't show tips at start up" IsChecked="{x:Bind HidingTips, Mode=TwoWay}" />
+                    <TextBlock>You can change your tip preferences in <Hyperlink NavigateUri="app:/item/SettingsPage">Settings</Hyperlink> if you change your mind.</TextBlock>
+                </StackPanel>
+        </muxc:TeachingTip>
+    </Button.Resources>
+</Button>
+```
+
+![A sample app with a teaching tip pointing at the save button. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." In the content area of the teaching tip is a CheckBox labeled "Don't show tips at startup" and underneath is text that reads "You can change your tip preferences in Settings if you change your mind" where "Settings" is a link to the app's settings page. There is a close button on the top right corner of the teaching tip.](TeachingTipContentSample.jpg)
```

Comment:
In this example, the Close button gets the "default button" treatment. But in the appendix ("Action button" section), the Action button gets the "default button" treatment. Does the app get to control which button looks default-like?

Reply:
Default button property was cut. All PM art will be updated with real screenshots shortly.